packages/text-input-react/src/TextArea.test.tsx

```diff
@@ -20,110 +20,6 @@ describe("TextArea", () => {
     });
 });
 
-describe("TextArea with autoExpand", () => {
```

Comment:
Hvorfor er alle testene borte?

Reply:
Nå regner vi ut antall rader for i `autoExpand` basert på `scrollHeight` til `<textarea>`. Den er alltid satt til 0 for JSDOM og her hadde vi trengt å kjøre testene i en ekte browser og ikke bruke JSDOM.

Reply:
Bra forklaring :) og ny god grunn til å få på plass et mer robust testregimet i ekte browsers.